### ISSUE TITLE
Filter for priority

### ISSUE THREAD

**Issue by hobarrera** [pimutils/todoman#19]
This issue was moved from the issue tracker at GitLab.

---

`todo list` should be able to filter by priority, i.e.:

`todo list --priority=high,medium`

or 

`todo list --priority=!low` 

---

`todo list --urgent` partially covers this. Further support requires supporting more granular priorities.

**Comment by untitaker** [comment]
How about making this work like log levels? Set the minimum priority to show, not list them.

**Comment by mansimarkaur** [comment]
I'd like to work on this issue.

**Comment by mansimarkaur** [comment]
Do we still want to keep `--urgent` if we want to keep `--priority=high`?
If we want to keep both, how will both of these be different?
Also, @untitaker  @hobarrera  what do you mean by setting it to log levels and what is the difference in showing and listing todos?

**Comment by untitaker** [comment]
Hello @mansimarkaur, glad to hear that!

>Do we still want to keep --urgent if we want to keep --priority=high?

At best it will become a shortcut for the latter, so I think we can remove it.

>setting it to log levels

I meant that `--priority=medium` should list anything that has a priority of `medium` or higher. I made an analogy to e.g. the `--verbosity` setting todoman has, which works similarly: If I set `--verbosity warning`, `warning`, `error` and `critical` log entries are shown (because they are more or equally important than/as `warning`), but `debug` entries are not.

>what is the difference in showing and listing todos?

Yeah it took me a while to understand my own sentence there ^^ I *think* I didn't mean "list todos", but rather that one shouldn't have to list all *priorities* for which tasks should be shown (only the minimum priority). If I didn't mean that, I guess it's complete nonsense and can be ignored :D
